concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/15 Implement Navigation.md

```diff
@@ -194,6 +218,72 @@ In this tutorial, we use the **List**:
 
 ##### Vue
 
+1. **Install <a href="https://router.vuejs.org/" target="_blank">Vue Router</a>**
+
+        npm install vue-router
+
+1. **Configure routing**
+
+    Specify routes and set up the router in the `main.js` file.
+
+1. **Define an [itemTemplate](/Documentation/ApiReference/UI_Widgets/dxList/Configuration/#itemTemplate)**
+
+    Specify the elements that the template should render and wrap them in a <a href="https://router.vuejs.org/api/#router-link" target="_blank">`<router-link>`</a>. In the code below, the *"links"* **itemTemplate** renders an icon and a text.
```

Comment:
```suggestion
    Specify the elements that the template should render and wrap them in a <a href="https://router.vuejs.org/api/#router-link" target="_blank">`<router-link>`</a>. In the code below, the *"links"* **itemTemplate** renders an icon and text.
```

Reply:
Is <router-link> a component? If so, would it be better to say `wrap them in a <a href="https://router.vuejs.org/api/#router-link" target="_blank">`<router-link>`</a> component.`